src/ioabc.cpp

```diff
@@ -959,6 +878,81 @@ void ABCInput::CreateWorkEntry()
     }
 }
 
+void ABCInput::FlushControlElements(Score *score, Section *section)
+{
+    Layer *layer = NULL;
+    Measure *measure = NULL;
+    for (auto iter = m_controlElements.begin(); iter != m_controlElements.end(); ++iter) {
+        if (!measure || (layer && layer->GetUuid() != iter->first)) {
+            layer = dynamic_cast<Layer *>(section->FindDescendantByUuid(iter->first));
+        }
+        if (!layer) {
+            LogWarning("ABC import: Element '%s' could not be assigned to layer '%s'",
+                iter->second->GetClassName().c_str(), iter->first.c_str());
+            delete iter->second;
+            continue;
```

Comment:
After the delete the pointer is dangling.  I'm not sure of the logic, suggest
iter->second = NULL;